Assets/Scenes/TutorialLevel.unity

```diff
@@ -15819,7 +15732,7 @@ MonoBehaviour:
   cutsceneController_: {fileID: 984060224}
   unitGridCombatArray_:
   - {fileID: 2046354194}
-  - {fileID: 1820880696}
+  - {fileID: 0}
```

Comment:
`fileID: 0` ??

Reply:
Good catch, this was the second Adam, I'll fix this.